docs/DTW_MP_trial.ipynb

```diff
@@ -0,0 +1,810 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #43.        return np.sqrt(cost[k_])
</pre><p>I recommend returning the squared cost rather than the actual cost as performing the sqrt here is very expensive. You can still compare squared costs against each other</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/733/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>